taar/recommenders/recommendation_manager.py

```diff
@@ -38,4 +38,11 @@ def recommend(self, client_id, limit):
         # Compute the recommendation.
         for r in self.recommenders:
             if r.can_recommend(client_info):
-                return r.recommend(client_info, limit)
+                logger.info("Using '{}' for client '{}'"
```

Comment:
Let's use the `extra` argument of the log function to store explicit values:
```python
logger.info("Recommender selected", extra={"client_id": client_id, "recommender": recommender_name})
```
This will result in a log line containing:
```json
{"message": "Recommender selected", "client_id": "<client id>", "recommender": "<recommender name>"}
```